src/panels/lovelace/cards/hui-entity-card.ts

```diff
@@ -183,6 +194,16 @@ export class HuiEntityCard extends LitElement implements LovelaceCard {
     fireEvent(this, "hass-more-info", { entityId: this._config!.entity });
   }
 
+  private _computeColor(numberValue: number | string): string {
```

Comment:
Why can `numberValue` be a string?

Reply:
Not all states are numbers

Reply:
Then we should skip this function right?

Reply:
wondering if I should bring over the state_color changes into this instead? much more flexible and would allow matching on either.